homework2/task4.py

```diff
@@ -0,0 +1,16 @@
+# Задайте список из N элементов, заполненных числами из промежутка [-N, N]. Найдите произведение элементов на указанных позициях. Позиции хранятся в файле file.txt в одной строке одно число.
+
+from random import randint
+
+N = 20
+numbers = []
+for i in range(N):
+    numbers.append(randint(-N, N))
+print(numbers)
+file = open('file.txt', 'r')
+multiplication = 1
+for line in file:
+    if int(line) < len(numbers) - 1:
```

Comment:
```suggestion
    if int(line) < N:
```
"Простое лучше сложного"

Reply:
Кстати, хоть в вашем случаи такого не произойдёт, но если в файле были бы отрицательные числа и если бы минимальное число из файла было бы меньше минимального отрицательного индекса в списке0 произошла бы ошибка `IndexError`